pluginconfig/pluginconfig-311.yaml

```diff
@@ -265,14 +265,14 @@ versions:
       genevaTDAgent: osarpint.azurecr.io/acs/td-agent:master.20190228.1
       logAnalyticsAgent: mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod03022020
       metricsServer: registry.access.redhat.com/openshift3/ose-metrics-server:v3.11.200
-      azureControllers: quay.io/openshift-on-azure/ci-azure:latest
```

Comment:
There's no new VM image/OpenShift version? We need to include that for security fixes.

Reply:
I wanted to play it safe :) updated and started new image publish as we didn't had one